### PR TITLE
Read join tokens from file, fixes #2515.

### LINKED ISSUE #2515: Support use of a path for auth_token in teleport.yaml
**What happened**:
Using a file path in the `auth_token` section of `teleport.yaml` doesn't read the token from the file - the string seems to be interpreted literally as being the token itself.

**What you expected to happen**:
Given that we support using a path in the `tokens` section under `auth_service` (https://gravitational.com/teleport/docs/admin-guide/#static-tokens), I would expect that providing a path to `auth_token` would read the token from the file in the same way.

**How to reproduce it (as minimally and precisely as possible)**:
`teleport.yaml` (node) - token is not read from the file:
```
teleport:
  auth_token: /var/lib/tokens/node/join-token
```

`teleport.yaml` (proxy/auth) - token is read from the file:
```
auth_service:
  enabled: true
  tokens:
    - "node:/var/lib/tokens/node/join-token"
```

**Environment**:
- Teleport version (use `teleport version`): `Teleport Enterprise v3.1.1git:v3.1.1-0-g8c506f47 go1.11.4
`
- OS (e.g. from /etc/os-release): Ubuntu 18.10

### PR DESCRIPTION
**Pull request by klizhentas**
Fixes #2515

### CODE REVIEW

**Review by webvictim — APPROVED** [review]
lgtm

**Comment by klizhentas** [comment]
retest this please

File: lib/utils/token.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	"strings"
+
+	"github.com/gravitational/trace"
+)
+
+// ReadToken is a utility function to read the token
+// from the disk if it looks like a path,
+// otherwise, treat it as a value
+func ReadToken(token string) (string, error) {
+	if !strings.HasPrefix(token, "/") {
```
Comment (fspmarshall) [inline lib/utils/token.go]: Do we support relative paths for the other config values?  If so, might be nice to support both `"/"` and `"./"`.

**Review by fspmarshall — APPROVED** [review]

File: lib/utils/token.go (hunk 1 above)
Comment (klizhentas) [reply lib/utils/token.go]: we don't support relative paths yet I think, and probably a good thing as we will avoid ambiguity here

**Review by russjones — APPROVED** [review]

**Pull request merged**